tests/test_cli/test_add/test_protocol.py

```diff
@@ -44,7 +44,7 @@ def setup_class(cls):
         cls.agent_name = "myagent"
         cls.cwd = os.getcwd()
         cls.t = tempfile.mkdtemp()
-        cls.protocol_name = "oef"
+        cls.protocol_name = "gym"
```

Comment:
Note, due to the dependencies now being installed for the connections automatically (and since the default connection is oef with protocol oef & fipa dependency) this had to be changed to a protocol different from oef, default and fipa